1-Authentication/0-sign-in-vanillajs/README.md

```diff
@@ -63,8 +63,8 @@ Here you'll learn about [ID Tokens](https://docs.microsoft.com/azure/active-dire
 * [Visual Studio Code](https://code.visualstudio.com/download) is recommended for running and editing this sample.
 * [VS Code Azure Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-node-azure-pack) extension is recommended for interacting with Azure through VS Code Interface.
 * A modern web browser.
-* An **Microsoft Entra External ID** tenant. For more information, see: [How to get an external tenant](https://github.com/microsoft/entra-previews/blob/PP2/docs/1-Create-a-CIAM-tenant.md)
-* A user account with permissions to create applications in your **Microsoft Entra External ID** tenant.
+* An **external Microsoft Entra ID** tenant. For more information, see: [Create a new tenant in Microsoft Entra ID](https://learn.microsoft.com/en-us/entra/fundamentals/create-new-tenant)
```

Comment:
I'll defer to @kengaderdus to say whether "external Microsoft Entra ID tenant" or "Microsoft Entra External ID tenant" is preferred

Reply:
Oh nevermind, I see Katherine asked for it this way 👍

Reply:
Just say _external tenant_, here and in all other instances as you already told the user the product name, that's _Microsoft Entra External ID._